modules/fontrendering/ext/freetype/CMakeLists.txt

```diff
@@ -1,79 +1,17 @@
-project(freetype)
+set(BUILD_SHARED_LIBS ON)
```

Comment:
?? What if it we set it to off in the cmake-gui?

Reply:
Then we have a problem since `freetype.lib` is not created. It is exposed by the freetype CMakeLists.txt anyway... Not the nicest thing, I know.

Reply:
We also want to build statically so this needs fixing

Reply:
Tested it again on a clean build. This line can be removed.